### PR TITLE
fix: update avatar to use new variable and docs standards

### PR DESCRIPTION
**Pull request by helendurranttw**
## 🖼 Context

The avatar was implemented before we had a coherent variable strategy for Neptune Web.

## 🚀 Changes

Tided up use of variables inside Avatar, and changed the docs in line with our standards.

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

**Comment by andreapiras** [comment]
eslint test are failing

File: packages/components/src/avatar/Avatar.js [hunk 1]
```diff
@@ -6,22 +6,32 @@ import { Size, Theme } from '../common';
 
 import './Avatar.css';
 
-const Avatar = ({ size, type, children, outlined, theme, backgroundColor, className }) => (
+const Avatar = ({ backgroundColor, children, className, outlined, size, theme, type }) => (
```
Comment (andreapiras) [inline packages/components/src/avatar/Avatar.js:9]: 😂

File: packages/components/src/avatar/Avatar.less [hunk 2]
```diff
@@ -35,24 +17,24 @@
     align-items: center;
```
Comment (andreapiras) [inline packages/components/src/avatar/Avatar.less:17]: Probably out of scope but this css can be replaced by display utilities

**Review by andreapiras — APPROVED** [review]

File: packages/components/src/avatar/Avatar.js (hunk 1 above)
Comment (helendurranttw) [reply packages/components/src/avatar/Avatar.js:9]: SHUT UP 😭

**Pull request merged**